src/remediations/controller.write.js

```diff
@@ -113,6 +132,12 @@ exports.create = errors.async(async function (req, res) {
     const id = uuid.v4();
 
     const result = await db.s.transaction(async transaction => {
+        if ( !_.isUndefined(name) ) {
+            await validateRemediationName(name, req.user.tenant_org_id, req.user.username, transaction);
+        } else {
+            throw new errors.BadRequest('UNDEFINED_NAME', 'Remediation name must be defined upon creation');
+        }
+
```

Comment:
When creating a new remediation, I think it will be necessary to throw this error when the name is undefined because this would result in a `null` name which we don't want.